### PR TITLE
add definitions file

### PR DESCRIPTION
**Pull request by philbrookes**

### CODE REVIEW

File: definitions.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Definition of Common Terms
+
+## Infrastructure
+
+- <a name="hub"/>**Hub**:
+    - **Definition**: A kubernetes cluster that is running OCM, typically connected to one or more [spokes](#spoke).
```
Comment (david-martin) [inline definitions.md:6]: What about linking out to https://multicluster.sigs.k8s.io/concepts/work-api/, which defines some of these terms, and give OCM as an example implementation that kuadrant uses.

File: definitions.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        - Application
+        - API
+        - deployment
+        - backend
+- <a name="gateway"/>**Gateway**:
+    - **Definition**:
+        - On a [spoke](#spoke): A resource designed to facilitate ingress to a [workload](#workload)
+        - On a [hub](#hub): A resource which follows specific mutation and replication rules to be replicated into [spokes](spokes).
+
+## Personas
```
Comment (david-martin) [inline definitions.md:30]: I think we should align with the existing personas at https://gateway-api.sigs.k8s.io/#personas,
and supplement them with the Platform Engineer persona, who's concern lies beyond a single cluster

File: definitions.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+- <a name="gateway-admin"/>**Gateway Admin**
+    - **Definition**: A user who is primarily concerned with configuring gateways across one or more [spokes](#spoke)
+    - Common alternatives to avoid:
+        - Sysadmin
+        - Admin
+        - SRE
+        - cluster-admin
+- <a name="cluster-admin" />**Cluster admin**:
+    - **Definition**: A user who is primarily concerned with the configuration of a particular [workload](#workload) connected to a [gateway](#gateway) on a [spoke](#spoke).
```
Comment (david-martin) [inline definitions.md:39]: > configuration of a particular [workload]

This sounds like the concern of the application developer.
Would the cluster operator persona in gateway api docs align with what you're thinking for cluster admin?

File: definitions.md (hunk 2 above)
Comment (philbrookes) [reply definitions.md:30]: this is a great suggestion, thanks Dave, I'll do this.

File: definitions.md (hunk 3 above)
Comment (philbrookes) [reply definitions.md:39]: yes, ok. I meant this more as "how a workload is connected to a gateway" and maybe need to add some messaging around "ensuring traffic is flowing to the workload correctly"

File: definitions.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+- <a name="spoke"/>**Spoke**:
+    - **Definition**: A Kubernetes cluster connected to a [hub](#hub) where resources can be placed via a manifestwork.
+    - Common alternatives to avoid:
+        - Workload cluster
+        - Cluster
+        - Gateway cluster
+        - Data plane
+        - Managed cluster
+- <a name="workload"/>**Workload**:
+    - **Definition**: A set of resources deployed to one or more [spokes](#spoke) - typically accessed via a [gateway](#gateway) - and designed to complete a task, e.g. an API or web-page.
```
Comment (pehala) [inline definitions.md:19]: Does this include HTTPRoutes?

File: definitions.md (hunk 1 above)
Comment (maleck13) [inline definitions.md:6]:
```suggestion
    - **Definition**: A kubernetes cluster that is running the OCM control plane components, typically connected to one or more [spokes](#spoke).
```

File: definitions.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+## Infrastructure
+
+- <a name="hub"/>**Hub**:
+    - **Definition**: A kubernetes cluster that is running OCM, typically connected to one or more [spokes](#spoke).
+    - Common alternatives to avoid:
+        - Control plane
+        - Control cluster
+- <a name="spoke"/>**Spoke**:
+    - **Definition**: A Kubernetes cluster connected to a [hub](#hub) where resources can be placed via a manifestwork.
```
Comment (maleck13) [inline definitions.md:11]: not sure I fully agree here a Spoke is referred to as a ManagedCluster even in OCM

```suggestion
    - **Definition**: A Kubernetes cluster connected to and managed by a [hub](#hub) where resources can be placed via a manifestwork.
```

File: definitions.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+    - Common alternatives to avoid:
+        - Workload cluster
+        - Cluster
+        - Gateway cluster
+        - Data plane
+        - Managed cluster
```
Comment (maleck13) [inline definitions.md:17]:
```suggestion
        - ~~Managed cluster~~
```

File: definitions.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+    - **Definition**: A set of resources deployed to one or more [spokes](#spoke) - typically accessed via a [gateway](#gateway) - and designed to complete a task, e.g. an API or web-page.
+    - Common Alternatives to avoid:
+        - Application
+        - API
+        - deployment
+        - backend
+- <a name="gateway"/>**Gateway**:
+    - **Definition**:
+        - On a [spoke](#spoke): A resource designed to facilitate ingress to a [workload](#workload)
+        - On a [hub](#hub): A resource which follows specific mutation and replication rules to be replicated into [spokes](spokes).
```
Comment (maleck13) [inline definitions.md:28]:
```suggestion
        - On a [hub](#hub): A resource that defines a Gateway to be replicated into [spokes](spokes).
```

**Comment by maleck13** [comment]
@philbrookes I think this is quite out of date. Closing , feel free to re-open

**Pull request closed without merging**